apps/emqx_bridge_pgsql/test/emqx_bridge_pgsql_SUITE.erl

```diff
@@ -147,7 +147,7 @@ common_init(Config0) ->
             ProxyPort = list_to_integer(os:getenv("PROXY_PORT", "8474")),
             emqx_common_test_helpers:reset_proxy(ProxyHost, ProxyPort),
             % Ensure enterprise bridge module is loaded
-            ok = emqx_common_test_helpers:start_apps([emqx_conf, emqx_bridge]),
+            ok = emqx_common_test_helpers:start_apps([emqx, emqx_postgresql, emqx_conf, emqx_bridge]),
```

Comment:
Nit: maybe we could start using `emqx_cth_suite` since we're updating this suite.

Reply:
We can do that in another PR